api/core/model_runtime/model_providers/google/llm/llm.py

```diff
@@ -204,6 +204,7 @@ def _generate(self, model: str, credentials: dict,
             stream=stream,
             safety_settings=safety_settings,
             tools=self._convert_tools_to_glm_tool(tools) if tools else None,
+            request_options={"timeout": 600}
```

Comment:
Please make this as an env var. If I remember this correctly, this should have already defined somewhere.